pkgs/development/compilers/elm/packages/ghc8_10/elm-instrument/default.nix

```diff
@@ -37,7 +27,7 @@ mkDerivation {
     quickcheck-io split tasty tasty-golden tasty-hunit tasty-quickcheck
     text
   ];
-  homepage = "https://elm-lang.org";
+  homepage = "http://elm-lang.org";
```

Comment:
what is the reason for this change? I think http version will redirect to https anyway.

Reply:
Cabal generated it. We're overriding this in a later stage.